--- /dev/null
+++ b/Mathlib/Unsorry/FourthPowerModFive.lean
@@ -0,0 +1,10 @@
+/-
+Copyright (c) 2026 Chris Barlow. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Barlow
+-/
+
+
+theorem fourth_power_mod_five (n : ℕ) (h : n % 5 ≠ 0) : n ^ 4 % 5 = 1 := by
+  rw [fourth_power_mod_five_reduce]
+  exact fourth_power_residue_mod_five (n % 5) (by omega) (by omega)
